### PR TITLE
Google Map

### PR DESCRIPTION
**Pull request by nrvpatel03**
Created the map with markers showing nearby locations of girlscout sellers with markers. Each marker has a popup window that shows their name, address, email and a link to their personal site page. Used the Google Api in order to accomplish this.

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ module.exports = {
 
         development: {
           username: "root",
-          password: process.env.DB_PASSWORD,
-          database: "girlScoutStore_DB",
+          password: "root",
```
Comment (nmclear) [inline config/config.js]: changed back to env password and girlScoutStore_DB

File: views/layouts/main.handlebars [hunk 2]
```diff
@@ -83,8 +83,12 @@
     integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
     crossorigin="anonymous"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/js/materialize.min.js"></script>
+<<<<<<< HEAD
+    {{!-- <script src="../../public/assets/javascript/materialize.js"></script> --}}
+=======
     <script  type="text/javascript" src="/assets/javascript/materialize.js"></script>
     <script  type="text/javascript" src="/assets/javascript/myCart.js"></script>
+>>>>>>> origin/master
```
Comment (yeungpirate) [inline views/layouts/main.handlebars]: We should look into cleaning up the comments regarding the conflicts.

File: views/landing.handlebars [hunk 3]
```diff
@@ -1,19 +1,125 @@
...
+        geocoder.geocode({'location': {lat: parseFloat(latLngArr[i].lat), lng: parseFloat(latLngArr[i].lng)}}, function(results, status){
+            if(status === 'OK'){
+                if(results[0]){
+                    genMarkers(namesArr[i],emailArr[i],results[0].formatted_address,{lat: parseFloat(latLngArr[i].lat), lng: parseFloat(latLngArr[i].lng)},idArr[i],map,markers);
+                }
+            }
+        });
+    }
+}
+function genMarkers(name,email,address, latlng,id, map,markers){
```
Comment (yeungpirate) [inline views/landing.handlebars]: Code is fine but format is off where you have a space after some commas, but not all.

File: views/layouts/main.handlebars [hunk 4]
```diff
@@ -83,8 +83,12 @@
...
```
Comment (nmclear) [inline views/layouts/main.handlebars]: Remove conflict comments

**Pull request merged**